client/Dockerfile

```diff
@@ -1,15 +1,6 @@
-#
-# Build the docker image
-# $ docker build -t user/dcrdex -f client/Dockerfile .
-#
-# Create docker volume to store client data
-# $ docker volume create --name=dcrdex_data
```

Comment:
Why remove the build and volume create commands?  Were they incorrect?

Reply:
removed in favor of this link - https://github.com/LasTshaMAN/dcrdex/blob/607675366f8116e01e05dd755883a4cec6940bc4/client/Dockerfile#L1-L1

thought it's not necessary to duplicate them in Docker file, and instructions themselves needed an update (after I've changed `CMD`->`ENTRYPOINT` existing `docker` commands need to pass arguments to `dexc` now)

Reply:
I had seen the readme change, but I guess I didn't understand what happened to the `volume create`.  Apologies, I don't use docker

Reply:
regarding `volume create`, 

it seems `-v dcrdex_data:/root/.dexc` covers volume creation in `docker run`.